packages/DataTable/src/components/Navigation/components/Sort/Sort.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import DropdownMenu from "@paprika/dropdown-menu";
+import sort from "../../../../helpers/sort";
+import { actions } from "../../../../constants";
+
+import { useDispatch, useDataTableState } from "../../../..";
+
+const propTypes = {
+  ColumnsDefinition: PropTypes.arrayOf(
+    PropTypes.shape({
+      props: PropTypes.object.isRequired,
+      type: PropTypes.func.isRequired,
+    })
+  ),
+};
+const defaultProps = { ColumnsDefinition: [] };
+
+const noop = () => {};
+
+export default function Sort(props) {
+  const dispatch = useDispatch();
+  const { sortColumn, sortDirection } = useDataTableState();
+  const { ColumnsDefinition } = props;
+  const hasSortDirections = ColumnsDefinition.find(
+    ({ props: columnProp }) => columnProp.sortDirections && columnProp.sortDirections.length > 0
+  );
+
+  function handleSort(columnId, direction) {
+    return () => dispatch({ type: actions.SORT, payload: { columnId, direction } });
+  }
+
+  if (!hasSortDirections) return null;
+
+  return (
+    <DropdownMenu
+      align="bottom"
+      renderTrigger={({ isOpen, handleOpenMenu }) => (
+        <DropdownMenu.Trigger isOpen={isOpen} onOpenMenu={handleOpenMenu}>
+          {sortColumn && sortDirection ? `Sort ${sortColumn} by ${sortDirection}` : "Sort"}
+        </DropdownMenu.Trigger>
+      )}
+    >
+      {ColumnsDefinition.map(({ props: columnProp }) => {
+        const { id: columnId, header, sortDirections } = columnProp;
+
+        if (!sortDirections || sortDirections.length === 0) return null;
+
+        return (
+          <DropdownMenu.Item key={columnId} onClick={noop}>
```

Comment:
why do we need to have our own button? and our own onClick, didn't that's why we have `DropDown.Item` ?

Reply:
From my understanding, `DropDown.Item` is each item in the dropdown list which is the blue box here:
![image](https://user-images.githubusercontent.com/38733362/69564642-3a754200-0f68-11ea-970e-badf45fc0aac.png)
But the `Item` itself is not clickable, the small pink button on the right is clickable

Reply:
then I guess we don't need a menu, we can refactor it later when we are more sure about what UI we are going to have at the end. :)